sources/packages/api/src/helpers/utilfunctions.ts

```diff
@@ -1,11 +1,12 @@
-import { StudentConfirmInfo } from "../route-controllers/student/models/student.dto";
+import { StudentInfo } from "../route-controllers/student/models/student.dto";
 export default class Helper {
-  static mapAddressAttributes(fromObject: any, toObject: StudentConfirmInfo) {
-    toObject.addressLine1 = fromObject.addressLine1;
-    toObject.addressLine2 = fromObject.addressLine2;
-    toObject.city = fromObject.city;
-    toObject.provinceState = fromObject.province;
-    toObject.country = fromObject.country;
-    toObject.postalCode = fromObject.postalCode;
+  static mapAddressAttributes(fromObject: any, toObject: StudentInfo) {
```

Comment:
I think there is type available for StudentContact. Please check [here](https://raw.githubusercontent.com/bcgov/SIMS/main/sources/packages/api/src/types/studentContact.ts)

Reply:
Thats a type. In the controller we are using models. The current code merges information from two services and prepares a data transfer object using the class in the model folder.

Reply:
CHanged as discussed